.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "4.1.2"
+  - 7
```

Comment:
We need to change the version on `package.json` as well, so they all agree. Actually, I'd expect Travis to respect the version on `package.json`, so we could remove it from here. One less place to keep the Node version in sync.